solr/core/src/java/org/apache/solr/handler/admin/ColStatus.java

```diff
@@ -93,8 +93,14 @@ public void getColStatus(NamedList<Object> results) {
     if (withRawSizeSummary || withRawSizeDetails) {
       withRawSizeInfo = true;
     }
-    if (withFieldInfo || withSizeInfo) {
-      withSegments = true;
+    boolean getSegments = false;
```

Comment:
maybe kind of bikeshedding, but what about `requireSegments`?   Or `segmentsNeeded`?   `getSegments` kind of looks like a method to me!

Reply:
FWIW I think `getSegments` is named fine, but so are your suggestions.  I don't think I've ever confused a method for a variable/field because the syntax involved would be nonsensical if reversed.  The line commented on here would unfathomably refer to a method (`type varName = value`).  Lines of code referencing the var won't have calling parenthesis so wouldn't be a method call.  And would be used in boolean ways (`if (getSegments)`) not a more interesting structure.